src/EventListeners.ts

```diff
@@ -195,9 +196,10 @@ export const generateEventHandlers = () => {
     }
 
     //Part 4: Toggles
-    // I'm just addressing all global toggles here: toggle1 up to toggle35
-    for (let index = 0; index < 35; index++) {
-        DOMCacheGetOrSet(`toggle${index+1}`).addEventListener('click', () => toggleSettings(index))
+    // I'm just addressing all global toggles here
+    const toggles = Array.from<HTMLElement>(document.querySelectorAll('button[id^="toggle"][toggleid]'));
```

Comment:
I don't think you need the extra call to Array.from. You should be able to access indices of an HTMLCollection?